topics/fastapi/index.md

```diff
@@ -1,9 +1,15 @@
 ---
 display_name: FastAPI
+aliases: fastapi-api, fastapi-framework, python-api
 logo: fastapi.png
 short_description: FastAPI is a modern & fast web framework for building APIs with Python 3.7+ based on standard Python type hints.
 topic: fastapi
+related: api, python3, django, flask
 github_url: https://github.com/tiangolo/fastapi
 created_by: Sebastián Ramírez
 ---
-FastAPI is a high performance, easy to learn, fast to code, and ready for production Python 3.7+ framework. Its performance is on par with NodeJS and Go (thanks to Starlette and Pydantic). It's intuitive and standards-based (based on API standards like OpenAPI (previously known as Swagger) and JSON Schema).
+
+**FastAPI** is a high-performance, easy-to-learn, fast-to-code Python 3.7+ framework ready for production. It boasts performance on par with NodeJS and Go, thanks to Starlette and Pydantic. FastAPI is intuitive and standards-based, following API standards like OpenAPI (previously known as Swagger) and JSON Schema.
+
+# Other resources
+- For more information, visit the [official FastAPI GitHub repository](https://github.com/tiangolo/fastapi)
```

Comment:
This link will be created automatically due to the `github_url` field, so it is redundant.